kqueen/engines/gce.py

```diff
@@ -189,6 +204,55 @@ def resize(self, node_count, **kwargs):
 
         return True, None
 
+    def set_network_policy(self, network_provider='CALICO', enabled=False):
```

Comment:
If I understand correctly what the method does, then you are not setting network policy in this method, you are enabling the possibility to set network policies, so I guess I would rename the method to `enable_network_policy`

Reply:
no, ideally this method can disable network policy too